Cargo.toml

```diff
@@ -106,8 +107,7 @@ rand_core = "0.6"
 rand_isaac = "0.3"
 rayon = "1"
 regex = "1.11"
-ring = "0.17.8"
-rustls = { version = "0.23", default-features = false, features = ["logging", "std", "tls12"] }
+rustls = { version = "0.23", features = ["logging", "std", "tls12"] }
```

Comment:
We should still turn off default features here. If an embedder wants to use ring, we don't want to prevent that.